site/_layouts/post.html

```diff
@@ -0,0 +1,80 @@
+<!-- Blog Layout Start -->
+{% assign abc = page.author | split: "," %}
+{% if abc.size > 1 %}
+{% assign img_link = abc.last %}
+{% else %}
+{% assign img_link = "http://pageanthive.com/images/empty-profile.png" %}
+{% endif %}
+{% assign name = abc.first %}
+<!DOCTYPE html>
+<html lang="{{ site.lang }}">
+
+  {% include head.html %}
+
+  <body id="page-top" data-spy="scroll" data-target=".navbar"  data-offset="151">
+
+    {% include navigation.html %}
+    {% if page.title == 'Blog' %}
+    <div class="container-fluid" id="topCont">
+      <div class="row">
+        <img src="{{ 'assets/images/blogTop.png' | relative_url }}">
+      </div>
+    </div>
+    {% endif %}
+
+    <div class="container" id="postsContainer">
+      <div class="row">
+    {% if page.title != 'Blog' %}
```

Comment:
Should we logically break this up into two layouts?

One for blogroll and one for a blogpost?

Reply:
Thought over it a bit, but isn't that way too small a piece of change to belong to it's own layout ?

Probably someday when someone comes to revamp the blog UI, they should consider this, but currently it only contains and additional image on top of a collection of summary of posts

Reply:
Cool. This could be a separate issue.